pkg/test/utils/kubernetes_integration_test.go

```diff
@@ -107,3 +110,56 @@ func TestClientWatch(t *testing.T) {
 
 	events.Stop()
 }
+
+func TestRunCommand(t *testing.T) {
```

Comment:
Would you please add tests making sure that hanging background processes are cleaned up at the end?